storage/docs/containers-storage-applydiff-using-staging-dir.md

```diff
@@ -18,7 +18,8 @@ Differently than **apply-diff**, the command **applydiff-using-staging-dir**
 first creates a staging directory and then moves the final result to the destination.
 
 ## EXAMPLE
-**containers-storage applydiff-using-staging-dir 5891b5b522 /path/to/diff**
+
+    containers-storage applydiff-using-staging-dir 5891b5b522 /path/to/diff
```

Comment:
I guess it would be best to simply put all of these in code blocks.

Reply:
I don't think it really matters either way. For short examples, indentation is a bit easier on the eye because less characters